examples/opengl_backend.py

```diff
@@ -0,0 +1,13 @@
+import taichi as ti
+
+ti.init(arch=ti.opengl)
```

Comment:
Having a whole file to serve for `ti.opengl` looks like an overkill. Can we just add a commented in, say, `mpm128.py`, 
```python
ti.init(arch=ti.cuda) # Try to run on GPU. Use arch=ti.opengl on old GPUs
```

Btw, let's bump the Taichi version to `0.6.0` and announce to the world after the OpenGL backend is fully ready :-) Just like what we did (`0.5.0`) for the Metal backend. Everybody will know that we have a new backend and we don't need an extra file in the examples.

Reply:
Good idea. We can write these comments after OpenGL is capable of mpm128 :) Btw the final GL backend can run but it randomly break down some time..

Reply:
Sounds good. Let's merge code in gradually and fix the breaking down issues one by one, before we announce the birth of the GL backend.